server/main.lua

```diff
@@ -33,6 +33,19 @@ QBX.Player_Buckets = {}
 QBX.Entity_Buckets = {}
 QBX.UsableItems = {}
 
+if GetConvar('qbx:acknowledge', 'false') ~= 'true' then
+    local welcomeMessage = [[^4
```

Comment:
Not sure if it would be in poor taste but maybe have it pull this from a repo to allow us to send messages to folks about important changes for updates?

Reply:
That's not a bad idea.

Reply:
Perhaps our `.github` repo would be appropriate?

Reply:
that or maybe txAdminRecipe